### PR TITLE
Update to 5.5

### PR DESCRIPTION
**Pull request by skylerfenn**

### CODE REVIEW

File: config/mail.php [hunk 1]
```diff
@@ -17,7 +17,6 @@
     */
 
     'driver' => env('MAIL_DRIVER', 'smtp'),
-    'override_wordpress' => true,
```
Comment (jeffberry) [inline config/mail.php]: Did you want to/intend to remove this?

File: gulpfile.js [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
- | file for our application, as well as publishing vendor resources.
- |
- */
-
-elixir(
-    function (mix)
-    {
-        mix.sass('app.scss', 'public/content/themes/larapress/assets/css');
-    }
-);
```
Comment (jeffberry) [inline gulpfile.js:19]: Did he finally get rid of Gulp?!

**Review by jeffberry — APPROVED** [review]

File: gulpfile.js (hunk 2 above)
Comment (skylerfenn) [reply gulpfile.js:19]: I think he did in 5.4 but I missed it.

File: config/mail.php (hunk 1 above)
Comment (skylerfenn) [reply config/mail.php]: No, good catch!

**Review by jeffberry — APPROVED** [review]

**Pull request merged**